layouts/services/single.html

```diff
@@ -41,9 +41,29 @@ <h4 class="text-center text-lg md:text-xl font-bold my-6">{{ .area }}</h4>
             </div>
         </section>
         {{ end }}
-        {{ end }}        
+        {{ end }}
+        
+        {{ if .testimonials.enable }}
+        {{ with .testimonials }}
 
-        {{ partial "services-testimonial.html" .}}
+        <section class="my-9 lg:my-14 grid md:grid-cols-2 gap-5">
+            {{ range .testimonial }}
+            <section class="bg-[#ECF6FF] p-6 rounded-xl">
+                <p class="text-xl text-center">
+                    {{ $quote := index (split .quote " ") }}
+                    <span class="font-bold text-[#A32C00]">{{ range first 2 $quote }} {{ . }} {{ end }}</span>
```

Comment:
This change will make it so that the first two words of any quote will be bold and have the color `#A32C00`.